src/fmu/sumo/sim2sumo/common.py

```diff
@@ -327,20 +236,16 @@ def __init__(
     ):
         self._logger = logging.getLogger(__name__ + ".Dispatcher")
         self._limit_percent = 0.5
-        self._parentid = get_case_uuid(datafile)
+        self._parentid = get_case_uuid(Path.cwd(), parent_level=1)
```

Comment:
Any reason for building a path here instead of passing it when the `Dispatcher` is created?
Not sure it matters, but I think it makes it a bit too specialized..

Reply:
Changed to `datafile.resolve()` in 38f71a0.